### ISSUE TITLE
Managing Climate Change with Blockchains and Oracles - Tecnalia/Chainlink Labs Report

### ISSUE THREAD

**Issue by ttaloute** [smartcontractresearchforum/engagement#146]
Link to Forum Post: https://www.smartcontractresearch.org/t/research-summary-managing-climate-change-with-blockchains-and-oracles-tecnalia-chainlink-labs-report/1509

**Comment by Fizzymidas** [comment]
Did this pass through the content pipeline?

**Comment by zubepaul** [comment]
@Fizzymidas No, it did not. This is a piece that was published external to SCRF but is getting included in the Discord 5 and we want to start tracking some metrics
